concrete/src/File/Set/Set.php

```diff
@@ -52,9 +52,11 @@ class Set
     public $fsName;
 
     /**
+     * @deprecated this field is not used anymore since ConcreteCMS 8
+     *
      * @var int
      */
-    //public $fsOverrideGlobalPermissions;
+    public $fsOverrideGlobalPermissions;
```

Comment:
Even if we don't use it anymore, there's a related database table column, so Concrete tries to assign this property to file set instances.